src/Init/Tactics.lean

```diff
@@ -207,6 +207,28 @@ the first matching constructor, or else fails.
 -/
 syntax (name := constructor) "constructor" : tactic
 
+/--
+Apply the first constructor,
+in the case that the goal is an inductive type with exactly two constructors.
+```
+example : True ∨ False := by
+  left
+  trivial
+```
```

Comment:
We decided a while back to standardize on the indicative mood in docstrings. I'm not great at remembering it, but I'm working on it!

```suggestion
Applies the first constructor when the goal is an inductive type with exactly two constructors, and fails otherwise.
```
example : True ∨ False := by
  left
  trivial
```
```

Reply:
I actually like the imperative one better, but I like consistency most of all!

I'll try to be aware of incoming changes, and we can chip away at it as we go.